torch_xla/core/c10d_registration.py

```diff
@@ -0,0 +1,25 @@
+import torch
```

Comment:
Since this is an implementation and not a new public API, can you put an underscore or move this to `_internal`?

Reply:
let me move this into _internal.